data/workflow.yaml

```diff
@@ -0,0 +1,300 @@
+-
+  name: dod_workflow
```

Comment:
Ah, I forgot - my plan was to let `human` default to `name`, with `_` replaced by actual spaces and the whole string set to Title Case.  Let me see if I can bake that into the django side, otherwise I'll include it in `readworkflow`.